flutter/test/android_platform_exception_event_processor_test.dart

```diff
@@ -0,0 +1,120 @@
+// ignore_for_file: invalid_use_of_internal_member
+
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:package_info_plus/package_info_plus.dart';
+import 'package:sentry_flutter/sentry_flutter.dart';
+import 'package:sentry_flutter/src/event_processor/platform_exception_event_processor.dart';
+
+import 'integrations/native_app_start_integration_test.dart';
+
+void main() {
+  late Fixture fixture;
+  setUp(() {
+    fixture = Fixture();
+
+    PackageInfo.setMockInitialValues(
```

Comment:
Since we can use this, we could actually remove the `PackageLoader` from our code, `setMockInitialValues` makes it testable anyway.

Reply:
https://github.com/getsentry/sentry-dart/issues/794